extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -42,11 +42,11 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
 
   private dragged: Element | undefined = undefined;
 
-  private googleContactsSearchEnabled: boolean;
+  private googleContactsSearchEnabled = false;
 
   constructor(view: ComposeView) {
     super(view);
-    this.googleContactsSearchEnabled = this.view.scopes.readContacts && this.view.scopes.readOtherContacts;
+    void this.initScopes();
```

Comment:
In this case, `googleContactsSearchEnabled` may remain wrongly `false` until the execution completes.
Do we have a unit test that relies on `googleContactsSearchEnabled=true`? It would fail if we introduce a delay in our '/oauth2/v1/tokeninfo' mock handler.
Therefore, I'd recommend to make `googleContactsSearchEnabled: Promise<boolean>` or `googleContactsSearchEnabled: boolean | Promise<boolean>`, so we could do something like `if (await this.googleContactsSearchEnabled)`,
but we'll have to make `renderSearchRes` function `async`
It'd be better

Reply:
FYU: it's ok to `await` a promise more than once, it will return the same value once resolved

Reply:
Nice suggestion 👍 
Thanks